### PR TITLE
Interfaces

### PR DESCRIPTION
**Pull request by Demi-AK**

### CODE REVIEW

**Comment by dkanghan** [comment]
can you rebase and use OVPS or other abbreviations where possible

**Review by Rutu136 — CHANGES_REQUESTED** [review]
Solve the merge conflict please.

File: SRS.md [hunk 1]
```diff
@@ -275,32 +276,91 @@ This section describes non-functional features of the software project. Specify
...
-The system shall automatically log out all customers after a period of inactivity. >
+The system shall automatically log out all customers after a period of inactivity. 
 
 ### 3.2.5 Interfaces
 
-##
+#### 3.2.5.1 User Interfaces
+- The Online Virtual Phone System shall provide a user-friendly web interface that allows users to make, receive, and manage virtual phone calls.
+- It should enable users to manage their account settings, view call logs, and utilize additional features such as voicemail, call forwarding, and conferencing.
+- The UI shall be accessible via web browsers and be responsive to ensure usability across various devices like computers, tablets, and smartphones.
```
Comment (Rutu136) [inline SRS.md]: I think we have not mentioned web browsers anywhere so I'd think desktop application.

File: SRS.md [hunk 2]
```diff
@@ -275,32 +276,91 @@ This section describes non-functional features of the software project. Specify
...
+
+#### 3.2.5.2 Software Interfaces
+- The system shall interface with various software components and external systems to facilitate its functionality. This might include interfacing with:
```
Comment (Rutu136) [inline SRS.md]: I'd say remove might and leave it as this shall

File: SRS.md [hunk 3]
```diff
@@ -275,32 +276,91 @@ This section describes non-functional features of the software project. Specify
...
+    * Billing Systems: to manage user subscriptions, usage billing, and payment processing.
+    * CRM Systems: to manage user information, support interactions, and user communication.
+    * External APIs: such as SMS gateways, Email systems, or third-party apps to enhance functionality. 
+
+#### 3.2.5.3 Hardware Interfaces
+- While the system is web-based and does not directly interact with hardware, it should be optimized for use on various devices with different hardware configurations.
```
Comment (Rutu136) [inline SRS.md]: You might have to change this based on our assumption that application is mobile and desktop.

File: SRS.md [hunk 4]
```diff
@@ -275,32 +276,91 @@ This section describes non-functional features of the software project. Specify
...
+- The system shall utilize VoIP (Voice over Internet Protocol) technologies to facilitate virtual phone calls.
+- It should securely transmit voice and data over the internet, ensuring clear, reliable, and secure communication.
+- Communication with external systems (like billing or CRM) should be done securely using API calls over HTTPS to ensure data integrity and security.
+
+#### 3.2.5.5 Third-party Interfaces
+- The system may interact with third-party services for specific functionalities (like SMS notifications, payment processing, or email communications).
+- Appropriate API integrations shall be established, ensuring secure and reliable data exchange with third-party systems.
+
+#### 3.2.5.6 Accessibility Interfaces
+- The system shall support 5 different languages such as English, Spanish, Hindi, and Mandrin.
```
Comment (Rutu136) [inline SRS.md]: it will be '4'.

File: SRS.md [hunk 5]
```diff
@@ -275,32 +276,91 @@ This section describes non-functional features of the software project. Specify
...
+- The system shall utilize a secure and reliable technology stack, which can efficiently manage real-time data related to virtual phone calls and support associated functionalities.
+
+### 3.3.3 User Interface Constraints
+- Considering the virtual nature of the phone system, the user interface shall be intuitive and user-friendly, ensuring users can navigate and operate the system with ease and minimal technical expertise. 
 
 ## 3.4 Legal, Copyright, and Other Notices
 
-###
+### 3.4.1 Legal Compliance
+- The E-Store shall adhere to all applicable local, state, and international laws and regulations related to e-commerce, data protection, and consumer protection.
```
Comment (Rutu136) [inline SRS.md]: It will OVPS instead of E-store

File: SRS.md [hunk 6]
```diff
@@ -275,32 +276,91 @@ This section describes non-functional features of the software project. Specify
...
-###
+### 3.4.1 Legal Compliance
+- The E-Store shall adhere to all applicable local, state, and international laws and regulations related to e-commerce, data protection, and consumer protection. 
+
+### 3.4.2 Copyright Notices
+- All content, including text, graphics, logos, and multimedia within the E-Store, should respect copyright laws and, where applicable, credit should be given to the rightful owners or appropriate licenses obtained.
+
+### 3.4.2 User Data Protection
+
+- The E-Store shall comply with global data protection regulations, such as the General Data Protection Regulation (GDPR) and the California Consumer Privacy Act (CCPA), to safeguard user data.
```
Comment (Rutu136) [inline SRS.md]: It will be OVPS instead of E-store

**Pull request merged**